src/application/services/useNoteSettings.ts

```diff
@@ -87,8 +94,14 @@ export default function (): UseNoteSettingsComposableState {
   const load = async (id: NoteId): Promise<void> => {
     noteSettings.value = await noteSettingsService.getNoteSettingsById(id);
     const response = await noteService.getNoteById(id);
+    const firstNoteBlock = response.note.content.blocks[0];
+    const limitCharsForNoteTitle = 50;
 
     parentNote.value = response.parentNote;
+
+    if (firstNoteBlock !== null && Boolean(firstNoteBlock.data.text)) {
```

Comment:
```suggestion
    if (firstNoteBlock && firstNoteBlock.data.text) {
```

Reply:
you can use the `hasValue()` util

Reply:
I cant do like this, because of eslint warnings

```
  105:9   error  Unexpected object value in conditional. The condition is always true                  @typescript-eslint/strict-boolean-expressions
  105:27  error  Unexpected any value in conditional. An explicit comparison or type cast is required  @typescript-eslint/strict-boolean-expressions
  ```